packages/pocl-job/src/transform/bindings/pocl/transaction/transaction.bindings.js

```diff
@@ -104,11 +107,13 @@ export const Transaction = new Binding({
     const paymentSource = dataSource === DIRECT_DEBIT_DATASOURCE ? DIRECT_DEBIT_PAYMENTSOURCE : POST_OFFICE_DATASOURCE
     const serialNumber = children[SerialNumber.element]
     const permit = children[licenceBindings.Permit.element]
+    const journalId = children[DDIDBinding.element]
 
     return {
       id: serialNumber,
       createTransactionPayload: {
         dataSource,
+        ...(journalId ? { journalId } : {}),
```

Comment:
Is the empty object necessary here? could this be changed to `...(journalId && {journalId})`